src/components/search/Search.test.tsx

```diff
@@ -0,0 +1,260 @@
+        }
+      }
+    ]
+  },
+  nodes: nodes,
+  handleSearchChange: jest.fn()
+};
+
+const { setup, spyOn } = composeComponentTestUtils(SearchBar, baseProps);
+
+describe(SearchBar.name, () => {
+  it('should render searchbar', () => {
+    const { baseElement } = render(<SearchBar {...baseProps} />);
+    expect(baseElement).toMatchSnapshot();
+  });
+  describe('handleInput()', () => {
+    it('should call handleInput', () => {
+      const { getByPlaceholderText } = render(<SearchBar {...baseProps} />);
+      const input = getByPlaceholderText('...');
+      expect(input).toBeTruthy();
+    });
+  });
+  describe('getAllNodes()', () => {
+    it('should call getAllNodes', () => {
+      const { instance } = setup(true);
+      const spy = spyOn('getAllNodes');
+      expect(spy).not.toHaveBeenCalled();
+      instance.getAllNodes();
+      expect(spy).toHaveBeenCalledTimes(1);
+    });
+    it('should return filtered nodes', () => {
+      const { instance } = setup(true);
+      const response = instance.getAllNodes();
+      expect(response).toEqual(filteredNode);
+    });
+  });
+  describe('findNodes()', () => {
+    it('should call findNodes', () => {
+      const { instance } = setup(true);
+      const spy = spyOn('findNodes');
+      expect(spy).not.toHaveBeenCalled();
+      instance.findNodes('teste');
+      expect(spy).toHaveBeenCalledTimes(1);
+    });
+    it('should return found nodes', () => {
+      const { instance } = setup(true);
+      const response = instance.findNodes('teste');
+      expect(response).toEqual(filteredNode);
+    });
+  });
+  describe('dragBackground()', () => {
+    it('should call dragBackground', () => {
+      const { instance } = setup(true);
+      const spy = spyOn('dragBackground');
+      expect(spy).not.toHaveBeenCalled();
+      instance.dragBackground();
+      expect(spy).toHaveBeenCalledTimes(1);
+    });
+  });
+  describe('toggleMoveSelected()', () => {
+    it('should call toggleMoveSelected', () => {
+      const { instance } = setup(true);
+      const spy = spyOn('toggleMoveSelected');
+      expect(spy).not.toHaveBeenCalled();
+      instance.toggleMoveSelected('up');
+      expect(spy).toHaveBeenCalledTimes(1);
+    });
+    it('should call dragBackground', () => {
+      const { instance } = setup(true);
+      const spy = spyOn('dragBackground');
+      expect(spy).toHaveBeenCalledTimes(2);
+      instance.toggleMoveSelected('up');
+      expect(spy).toHaveBeenCalledTimes(3);
+    });
+    it('should set search', () => {
+      const { instance, props } = setup(true);
+      instance.toggleMoveSelected('down');
+      expect(props.search).not.toEqual({});
+    });
+  });
+  describe('closeSearch()', () => {
+    it('should call closeSearch', () => {
+      const { instance } = setup(true);
+      const spy = spyOn('closeSearch');
+      expect(spy).not.toHaveBeenCalled();
+      instance.closeSearch();
+      expect(spy).toHaveBeenCalledTimes(1);
+    });
+  });
+});
```

Comment:
Adding spaces between `describe` functions will make the code a little more readable:

```suggestion
describe(SearchBar.name, () => {
  it('should render searchbar', () => {
    const { baseElement } = render(<SearchBar {...baseProps} />);
    expect(baseElement).toMatchSnapshot();
  });

  describe('handleInput()', () => {
    it('should call handleInput', () => {
      const { getByPlaceholderText } = render(<SearchBar {...baseProps} />);
      const input = getByPlaceholderText('...');
      expect(input).toBeTruthy();
    });
  });

  describe('getAllNodes()', () => {
    it('should call getAllNodes', () => {
      const { instance } = setup(true);
      const spy = spyOn('getAllNodes');
      expect(spy).not.toHaveBeenCalled();
      instance.getAllNodes();
      expect(spy).toHaveBeenCalledTimes(1);
    });
    it('should return filtered nodes', () => {
      const { instance } = setup(true);
      const response = instance.getAllNodes();
      expect(response).toEqual(filteredNode);
    });
  });

  describe('findNodes()', () => {
    it('should call findNodes', () => {
      const { instance } = setup(true);
      const spy = spyOn('findNodes');
      expect(spy).not.toHaveBeenCalled();
      instance.findNodes('teste');
      expect(spy).toHaveBeenCalledTimes(1);
    });
    it('should return found nodes', () => {
      const { instance } = setup(true);
      const response = instance.findNodes('teste');
      expect(response).toEqual(filteredNode);
    });
  });

  describe('dragBackground()', () => {
    it('should call dragBackground', () => {
      const { instance } = setup(true);
      const spy = spyOn('dragBackground');
      expect(spy).not.toHaveBeenCalled();
      instance.dragBackground();
      expect(spy).toHaveBeenCalledTimes(1);
    });
  });

  describe('toggleMoveSelected()', () => {
    it('should call toggleMoveSelected', () => {
      const { instance } = setup(true);
      const spy = spyOn('toggleMoveSelected');
      expect(spy).not.toHaveBeenCalled();
      instance.toggleMoveSelected('up');
      expect(spy).toHaveBeenCalledTimes(1);
    });
    it('should call dragBackground', () => {
      const { instance } = setup(true);
      const spy = spyOn('dragBackground');
      expect(spy).toHaveBeenCalledTimes(2);
      instance.toggleMoveSelected('up');
      expect(spy).toHaveBeenCalledTimes(3);
    });
    it('should set search', () => {
      const { instance, props } = setup(true);
      instance.toggleMoveSelected('down');
      expect(props.search).not.toEqual({});
    });
  });

  describe('closeSearch()', () => {
    it('should call closeSearch', () => {
      const { instance } = setup(true);
      const spy = spyOn('closeSearch');
      expect(spy).not.toHaveBeenCalled();
      instance.closeSearch();
      expect(spy).toHaveBeenCalledTimes(1);
    });
  });
});
```